platforms/documentation/docs/src/docs/userguide/authoring-builds/gradle-properties/service_injection.adoc

```diff
@@ -33,11 +34,14 @@ The following services are available for injection:
 7. <<archiveoperations,`ArchiveOperations`>> - Allows a task to run operations on archive files such as ZIP or TAR files.
 8. <<execoperations,`ExecOperations`>> - Allows a task to run external processes with dedicated support for running external `java` programs.
 9. <<toolingmodelbuilderregistry,`ToolingModelBuilderRegistry`>> - Allows a plugin to registers a Gradle tooling API model.
+10. <<testeventreporterfactory,`TestEventReporterFactory`>> - Allows a plugin to access Gradle's test events and its corresponding API.
```

Comment:
:question: Is `TestEventReporterFactory` available for injection in task actions and work actions? If not, this should be clarified.

Reply:
Sterling confirmed it will be available in the future. Removing for now.